### ISSUE TITLE
Cannot use 'new' with an expression whose type lacks a call or construct signature.

### ISSUE THREAD

**Issue by Ofer-Gal** [s-KaiNet/spfx-vue-hello-world#1]
For line with 
```
new Vue({
      el: `#app-${this.context.instanceId}`,
      data: this.data
    });

```
I get an Error
HelloWorldWebPart.ts(28,4): error TS2351: Cannot use 'new' with an expression whose type lacks a call or construct signature.
Maybe it is  a version issue

**Comment by s-KaiNet** [comment]
Hi, sorry for delayed response, was on vacation.   
You're right, I believe the repo needs to be updated, because a lot of new stuff have appeared since original version. I will take a look in nearest days.

**Comment by s-KaiNet** [comment]
Hi, first of all sorry for "nearest days" being expanded to almost two months.    
I've fixed issue, actually vue 2.5 introduces a few more difficulties even for hello world scenario. I updated blog post as well.
